### ISSUE TITLE
Windows App and Linux Subsystem Database interactions

### ISSUE THREAD

**Issue by sujaysarma** [Microsoft/BashOnWindows#1151]
Question / Feature request

* If I have an app on the Windows host and a database (say even a Json file) installed into the Linux Subsystem, how can the Win-hosted app access this database?

* Expected result -- Some form of path or connection string in the feature documentation?

* Actual results -- not sure, no idea if this is even possible.

* Your Windows build number -- 14393.222

**Comment by sundhaug92** [comment]
If it's network-based, yes. It even shares IP (and localhost).

**Comment by sujaysarma** [comment]
Could you give an example? For example, if a MySQL db is installed, it would be accessible as <windows-hostname:port> ?

**Comment by sujaysarma** [comment]
Ok, I got it by experimenting. I installed mysql-server on my WSL. Had to set it up to bind to all ip addresses (bind-address=0.0.0.0 in /etc/mysql/my.cnf and then service mysql restart). After opening port 3306 in the Windows Firewall (using Advance Security MMC), I am able to remote-connect to the instance from other boxes using "mysql -h <ipaddress> -u <user> -p". 

Of course, don't forget to grant remote login privs to the <user>!
